includes/content-distribution/class-api.php

```diff
@@ -133,7 +138,11 @@ public static function distribute( $request ) {
 			return new WP_Error( 'newspack_network_content_distribution_error', $distribution->get_error_message(), [ 'status' => 400 ] );
 		}
 
-		Content_Distribution_Class::distribute_post( $outgoing_post, $status_on_create );
+		$payload = $outgoing_post->get_payload( $status_on_create );
+		Data_Events::dispatch( 'network_post_updated', $payload );
+
+		// Store payload hash to prevent unnecessary updates.
+		update_post_meta( $post->ID, self::PAYLOAD_HASH_META, $outgoing_post->get_payload_hash( $payload ) );
```

Comment:
```suggestion
		update_post_meta( $post_id, Content_Distribution_Class::PAYLOAD_HASH_META, $outgoing_post->get_payload_hash( $payload ) );
```

This line was giving me a fatal error

Reply:
Whoops! Thank you for catching that.